nx-X11/programs/Xserver/include/os.h

```diff
@@ -460,6 +460,11 @@ extern _X_EXPORT size_t
 strlcat(char *dst, const char *src, size_t siz);
 #endif
 
+#ifndef HAVE_TIMINGSAFE_MEMCMP
```

Comment:
done. minus library handling. That should be done by someone who has such a system for building.